select/sdks/android/guide-v2.md

```diff
@@ -1,114 +1,87 @@
-# A guide for card enrollment with the iOS SDK
+# A guide for card enrollment with the Android SDK <a style="color: #111;" class="improve-docs" href="/select/sdks/android/guide-v1">v1</a> <a style="margin-right: auto; border-bottom: 2px solid #0048ff;" class="improve-docs" href="/select/sdks/android/guide-v2">v2</a>
```

Comment:
Not sure why git thinks that I moved the iOS file to this folder., but that will be fixed as I have 2 guides for each version for iOS coming anyway.